code/src/WijmoProvider/Features/Export.ts

```diff
@@ -45,10 +57,38 @@ namespace WijmoProvider.Feature {
             this._grid.features.pagination.changePageSize(0);
         }
 
+        private _showLoadingMessage() {
+            const gridId = this._grid.widgetId;
+            const parentPlaceholder = document.querySelector(
+                '#' +
+                    gridId +
+                    ' .' +
+                    OSFramework.Helper.Constants.gridAutogeneratedPlaceholderCss
+            );
+            const loadingPlaceholderContent = `<div class="datagrid-loading heading4 OSInline" style="width: 100%; height: 100%;"><i class="icon fa-spin fa fa-spinner fa-1x"></i><div class="OSInline" style="margin-left: 10px;">${this._loadingMessage}</div></div>`;
+            const createdDivElem = document.createElement('div');
+            createdDivElem.className =
+                OSFramework.Helper.Constants.overlayExportFeedbackCss;
+            createdDivElem.innerHTML = loadingPlaceholderContent;
```

Comment:
Using innerHTML here will not pass if a security audit is performed on this code. The usage of innerHTML is dangerous and can lead to cross-site scripting. At the moment, there is a experimental technology that is used to parse and sanitize a string of HTML and then insert it into the DOM as a subtree of the element - Element.setHTML(). But as I said, it's experimental.
There is another way of going around this: https://developer.mozilla.org/en-US/docs/Web/API/DOMParser/parseFromString. Text is sanitized and transformed into HTML element (if you specify that way), all you have to do is to append that element to the one you need (createdDivElement.append(element))

Reply:
Nice catch @joanabpereira 
@glundgren93 Can you please review this?

Reply:
I'll create a task to improve this in the future and we'll move forward with this